### PR TITLE
Add nfts function

### PR DESCRIPTION
**Pull request by AtlasSchmidt**
## implement action of nfts.
- get blog  
- search 
- scroll.

### CODE REVIEW

File: src/components/Blog/index.js [hunk 1]
```diff
@@ -7,6 +7,15 @@ import IconButton from '@mui/material/IconButton';
 import SearchIcon from '@mui/icons-material/Search';
 
 const Blog = () => {
+
```
Comment (codeLover1126) [inline src/components/Blog/index.js:10]: NEVER put a blank line at the beginning of the component.

File: src/components/Blog/index.js [hunk 2]
```diff
@@ -7,6 +7,15 @@ import IconButton from '@mui/material/IconButton';
...
+    const getBlog = async () => {
+        const res = await fetch(
+            `https://api-mainnet.magiceden.io/idxv2/getListedNftsByCollectionSymbol?collectionSymbol=okay_bears&limit=20&offset=${offset}`
+        )
+        const data = await res.json()
+        setFilterBlog([...blogdata, ...data.results])
+        console.log(res)
```
Comment (codeLover1126) [inline src/components/Blog/index.js:17]: Please get rid of all `console`s.

File: src/components/Blog/index.js [hunk 3]
```diff
@@ -7,6 +7,15 @@ import IconButton from '@mui/material/IconButton';
 import SearchIcon from '@mui/icons-material/Search';
 
 const Blog = () => {
+
+    const getBlog = async () => {
```
Comment (codeLover1126) [inline src/components/Blog/index.js:11]: Indentation size is 2, not 4.

File: src/components/Blog/index.js [hunk 4]
```diff
@@ -16,6 +16,9 @@ const Blog = () => {
         setFilterBlog([...blogdata, ...data.results])
         console.log(res)
     }
+    useEffect(() => {
```
Comment (codeLover1126) [inline src/components/Blog/index.js:19]: It's good to add a blank line above here and after this `useEffect` declaration.

**Review by codeLover1126 — CHANGES_REQUESTED** [review]
Nice job! Please make sure the indentation space is 2, not 4.

**Comment by codeLover1126** [comment]
Again, no description!

File: src/components/Blog/index.js (hunk 1 above)
Comment (AtlasSchmidt) [reply src/components/Blog/index.js:10]: this is resolved

File: src/components/Blog/index.js (hunk 2 above)
Comment (AtlasSchmidt) [reply src/components/Blog/index.js:17]: this is resolved

File: src/components/Blog/index.js (hunk 3 above)
Comment (AtlasSchmidt) [reply src/components/Blog/index.js:11]: sorry, this is resolved

File: src/components/Blog/index.js (hunk 4 above)
Comment (AtlasSchmidt) [reply src/components/Blog/index.js:19]: this is resolved

**Comment by codeLover1126** [comment]
Please pull from `features/nfts-ui` and resolve conflict.

**Pull request merged**